src/Polly.Core/Hedging/HedgingActionGeneratorArguments.cs

```diff
@@ -7,17 +7,20 @@ namespace Polly.Hedging;
 /// </summary>
 /// <typeparam name="TResult">The type of the result.</typeparam>
 /// <remarks>
+/// The <see cref="PrimaryContext"/> represents the context that was received by the hedging strategy and used to execute the primary action.
+/// To prevent race conditions, hedging strategy then clones the primary context into <see cref="ActionContext"/> and uses it to execute the hedged action.
+/// Every hedged action gets its own context that is cloned from primary.
```

Comment:
```suggestion
/// To prevent race conditions, the hedging strategy then clones the primary context into <see cref="ActionContext"/> and uses it to execute the hedged action.
/// Every hedged action gets its own context that is cloned from the primary.
```